### PR TITLE
Added parameters to `ebpf_tracing.cmd`, revised tracing/cleanup logic

### LINKED ISSUE #2372: Add named parameters to `ebpf_tracing.cmd`
### Describe the feature you'd like supported

All parameters that are currently hard-coded in the script, will need to be overridable by named parameters.

### Proposed solution

Parameters, along with their defaults will be:

- `command` (mandatory)
- `trace_path` (mandatory)
- `trace_name="ebpf_diag"`
- `rundown_period=0:35:00`
- `max_file_size_mb=20`
- `max_committed_etl_files=30`
- `max_committed_wfp_state_files=1`

### Additional context

This will be a precondition for #2365

### PR DESCRIPTION
**Pull request by gtrevi**
Closes #2372

## Description

- Update the tracing and cleanup logic.
- Added named parameters to `ebpf_tracing.cmd`.
- Parametrized scheduled tasks with newly introduced parameters.

## Testing

Local testing: 12h+ with occasional reboots in between (for automatic startup scenario).

## Documentation

n.a.

### CODE REVIEW

File: scripts/ebpf_tracing.cmd [hunk 1]
```diff
@@ -84,22 +133,33 @@ if "%option%"=="periodic" (
...
+echo   /trace_name name                     Name of the logman trace (Default: "ebpf_diag")
+echo   /rundown_period period               Period, expressed as (H:mm:ss) for saving and generating a new ETL log, and for generating a WFP state snapshot (Default: 0:35:00).
+echo   /max_file_size_mb size               Maximum size set for an ETL log (Default: 20).
+echo   /max_committed_etl_files count       Number of (most recent) .ETL files to keep in the main 'trace_path\committed' (Default: 30)
+echo   /max_committed_wfp_state_files count Number of (most recent) WFP-state .CAB files to keep in the main 'trace_path\committed' (Default: 1).
+echo Examples (overriding defaults):
+echo        ebpf_tracing.cmd start /trace_name ebpf_diag /trace_path "%SystemRoot%\Logs\eBPF" /rundown_period 0:35:00 /max_file_size_mb 20
+echo        ebpf_tracing.cmd stop /trace_name ebpf_diag /trace_path "%SystemRoot%\Logs\eBPF"
+echo        ebpf_tracing.cmd periodic periodic /trace_path "%SystemRoot%\Logs\eBPF" /max_file_size_mb 20 /max_committed_etl_files 30 /max_committed_wfp_state_files 1
```
Comment (matthewige) [inline scripts/ebpf_tracing.cmd:170]: I think there is an extra 'periodic' here

File: scripts/ebpf_tracing.cmd [hunk 2]
```diff
@@ -3,76 +3,131 @@
...
+@rem Uncomment ECHOs below for debugging purposes.
+@rem ----------------------------------------------
+@rem echo Running with the following parameter values:
+@rem echo command=%command%
+@rem echo trace_path=%trace_path%
+@rem echo trace_name=%trace_name%
+@rem echo rundown_period=%rundown_period%
+@rem echo max_file_size_mb=%max_file_size_mb%
+@rem echo max_committed_etl_files=%max_committed_etl_files%
+@rem echo max_committed_wfp_state_files=%max_committed_wfp_state_files%
```
Comment (matthewige) [inline scripts/ebpf_tracing.cmd:66]: Do you want to check this in, or was this just for development?

File: scripts/ebpf_tracing.cmd (hunk 1 above)
Comment (gtrevi) [reply scripts/ebpf_tracing.cmd:170]: corrected

File: scripts/ebpf_tracing.cmd (hunk 2 above)
Comment (matthewige) [reply scripts/ebpf_tracing.cmd:66]: Sounds good!

**Review by matthewige — DISMISSED** [review]

**Review by dthaler — DISMISSED** [review]

**Review by saxena-anurag — APPROVED** [review]

**Review by saxena-anurag — APPROVED** [review]

**Review by dthaler — APPROVED** [review]

**Pull request merged**